src/containers/Home/OurPartnersSection/ServiceTile/ServiceTile.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './ServiceTile.module.css';
+
+const ServiceTile = ({ children, src, title, url }) => {
+
+  return (
+    <div className={styles.services_tile}>
```

Comment:
```suggestion
    <div className={styles.services_tile}>
      <h4>{title}</h4>
      <img src={src} alt={title} />
      <p>{children}</p>
      <a href={url}>Scaled Agile Framework</a>
    </div>
```
The `<ServiceTile>` component receives the url, and then uses it for the value of the href property in the `<a>` element.

Reply:
When I tried that, I kept getting urls that looked like: localhost:3000/www.scrum.org.  Thank you for figuring that out.